app/src/main/AndroidManifest.xml

```diff
@@ -28,26 +31,19 @@
             android:name=".ui.SplashActivity"
             android:screenOrientation="portrait"
             android:windowSoftInputMode="adjustResize">
-
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-
         </activity>
-
         <activity
             android:name=".ui.VerifyPhoneActivity"
             android:screenOrientation="portrait"
-            android:windowSoftInputMode="adjustResize">
-
-        </activity>
-
+            android:windowSoftInputMode="adjustResize"></activity>
```

Comment:
Change to `android:windowSoftInputMode="adjustResize" />`. Doesn't it show warning?